index.js

```diff
@@ -4,6 +4,47 @@ var path = require('path');
 var mergeTrees = require('broccoli-merge-trees');
 var Funnel = require('broccoli-funnel');
 
+function toNodeJSBuilder(projectRoot) {
+  var esTranspiler = require('broccoli-babel-transpiler');
+  var broccoli = require('broccoli-builder');
+
+  var appEnvPath = path.join(projectRoot, 'config', 'environment.js');
+  var appEnv = require(appEnvPath);
+
+  let emberCliMiragePath = __dirname;
+  let mirageAddon = Funnel(emberCliMiragePath, {srcDir: 'addon', destDir: 'ember-cli-mirage'});
+  let appMirageConfig = Funnel(projectRoot, {srcDir:'mirage', destDir: 'mirage'});
```

Comment:
Mirage's dir can be configurable, we might need to look it up here... I kind of wish it wasn't though. shrug